onadata/libs/utils/export_builder.py

```diff
@@ -206,6 +206,22 @@ def track_task_progress(additions, total=None):
         pass
 
 
+def get_parent_element(xpath, element_type, data_dictionary):
+    parent_xpath = '/'.join(xpath.split('/')[:-1])
+    return data_dictionary.get_element(parent_xpath)
+
+
+def get_element_from_xpath(xpath, elements):
+    return next((el for el in elements if el['xpath'] == xpath), None)
+
+
+def get_element_type_from_xpath(xpath, elements):
+    element = get_element_from_xpath(xpath, elements)
+    if element:
+        return element['type']
+    return "unknown"
```

Comment:
"unknown" should be constant.

We probably should not return "unknown",  returning None or empty string would be more consistent elements in XForm dict.